docs/backup-modules.md

```diff
@@ -1,77 +1,64 @@
-# Backing up of existing modules
+# Backing up of existing modules (`--backup-modules`)
```

Comment:
I wouldn't mention `--backup-modules` in the title, since that results in line wrapping of the title on the rendered page.